### PR TITLE
Fix issue with repeatable flexible field

### PR DESCRIPTION
**Pull request by milewski**
Hi, 

I found an issue with this package and flexible content, somehow having multiple items when using flexible field cause a weird property to be saved alongside the original data... example:

image

What this PR does is instead of always removing the last `.*` from the attribute get whatever is before the first occurrence of `.*`

Currently this `header.*.*` returns `header.*`
With this PR `header.*.*` returns `header`

### CODE REVIEW

**Comment by Tarpsvo** [comment]
Hey! Flexible creator already submitted a different solution to this. Did you test the latest version? Thanks!

**Comment by milewski** [comment]
Oh I wasn't using the last version.. things move so fast here haha .. ok let me try again with his fix

**Comment by milewski** [comment]
Ok confirmed. no issue on the latest version thanks
